server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/webhook/WebhookCallerImpl.java

```diff
@@ -71,9 +74,17 @@ public WebhookDelivery call(Webhook webhook, WebhookPayload payload) {
   }
 
   private static Request buildHttpRequest(Webhook webhook, WebhookPayload payload) {
+    HttpUrl url = HttpUrl.parse(webhook.getUrl());
+    if (url == null) {
```

Comment:
outside the scope of the review: I suggest to add our own `checkArgument` methods